kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -31,7 +31,7 @@ function isExercise(o) {
 /**
  * Composable function presenting primary interface for Quiz Creation
  */
-export default function useQuizCreation(DEBUG = false) {
+export default function useQuizCreation(DEBUG = true) {
```

Comment:
This is the cause of the failing tests. Could you maybe update the line to this:

`export default function useQuizCreation(DEBUG = false /* do not commit with this set to true */) {`